### DISCUSSION TITLE
尝试使用 1.19.4-Forge<latest> 加载模组出错

### DISCUSSION THREAD

**Discussion by 8MiYile** [IndustrialCraftMC/IndustrialCraft2#7] [【漏洞反馈】Bugs Report]
旁加载：Forge & Optifine
Forge Mod List: 工业2

https://mclo.gs/0oYWtBp
https://mclo.gs/YOOEDhq
https://mclo.gs/OIvk3rq
三份日志来自HMCL生成
https://mclo.gs/mgvsAwF

游戏主菜单都没见到就崩了

**Comment by AbbottMc** [comment]
目前支持的是1.19.2版本，还没升级到1.19.4

**Reply by LrbsX** [reply]
如果升级了，之后会考虑给1.19.2一个可玩版本吗

**Reply by AbbottMc** [reply]
目前的计划是暂时只在1.19.2进行开发，先在高版本站稳脚跟再谈升级的事情
